doc/src/qwc_configuration.md

```diff
@@ -172,7 +172,7 @@ The first step is to prepare a QGIS project. Besides the common tasks of adding
 | Print layouts        | Layout manager                            | The print layouts offered in the Print plugin.   |
 | Print layout labels  | Layout manager                            | Print layout labels with an ID will be exposed in the Print plugin. Note: a label ID starting with `__` will not be exposed. |
 
-#### <a name="themesConfig-json"></a>Configuring the themes in `themesConfig.json`
```

Comment:
Hmm, these names are the anchors which are referenced above, why remove them? They are more stable than autogenerated anchors.

Reply:
I had removed them because they don't work with `hugo` so I assumed they just don't work at all.